documentation/modules/ROOT/pages/configuration/topic-auto-create-config.adoc

```diff
@@ -13,40 +13,96 @@
+* xref:configuring-automatic-topic-creation-in-kafka-connect[]
+* xref:configuration-of-automatically-created-topics[]
+* xref:debezium-connector-topic-creation-groups[]
+* xref:debezium-connector-topic-creation-group-configuration-properties[]
+* xref:specifying-the-configuration-for-the-debezium-default-topic-creation-group[]
+* xref:specifying-the-configurations-for-debezium-custom-topic-creation-groups[]
+* xref:registering-debezium-custom-topic-creation-groups[]
+endif::product[]
+
+// Type: procedure
+// ModuleID: disabling-automatic-topic-creation-for-the-kafka-broker
+[id="disabling-automatic-topic-creation-for-the-kafka-broker"]
+== Disabling automatic topic creation for the Kafka broker
+
+By default, the Kafka broker configuration enables the broker to create topics at runtime if the topics do not already exist.
+Topics created by the broker cannot be configured with custom properties.
+If you use a Kafka version earlier than 2.6.0, and you want to create topics with specific configurations, you must to disable automatic topic creation at the broker,
+and then explicitly create the topics, either manually, or through a custom deployment process.
+
+.Procedure
+
+* In the broker configuration, set the value of `auto.create.topics.enable` to `false`.
+
+// Type: procedure
+// ModuleID: configuring-automatic-topic-creation-in-kafka-connect
+// Title: Configuring automatic topic creation in Kafka Connect
+[id="enabling-automatic-topic-creation"]
 == Set up Kafka Connect
 
-Kafka Connect since Kafka 2.6.0 comes with topic creation enabled:
+Automatic topic creation in Kafka Connect is controlled by the `topic.creation.enable` property.
+The default value for the property is `true`, enabling automatic topic creation, as shown in the following example:
 
 [source,options="nowrap",shell]
 ----
 topic.creation.enable = true
 ----
 
+The setting for the `topic.creation.enable` property applies to all workers in the Connect cluster.
+
+Kafka Connect automatic topic creation requires you to define the configuration properties that Kafka Connect applies when creating topics.
+You specify topic configuration properties in the {prodname} connector configuration by defining topic groups, and then specifying the properties to apply to each group.
+The connector configuration defines a default topic creation group, and, optionally, one or more custom topic creation groups.
+Custom topic creation groups use lists of topic name patterns to specify the topics to which the group's settings apply.
+For details about how Kafka Connect matches topics to topic creation groups, see xref:topic-creation-groups[].
+For more information about how configuration properties are assigned to groups, see xref:topic-creation-group-configuration-properties[Topic creation group configuration properties].
+
+By default, topics that Kafka Connect creates are named based on the pattern `server.schema.table`, for example, `dbserver`.myschema.inventory`.
```

Comment:
not enough or too many ` `` ` signs?

Reply:
Oops. Yes. Thanks, for catching that ... one extra there. Fixed it.